src/components/context_menu/context_menu.tsx

```diff
@@ -68,10 +68,20 @@ export interface EuiContextMenuPanelDescriptor {
   initialFocusedItemIndex?: number;
 }
 
+export const sizeToClassNameMap: { [size in MenuSize]: string | null } = {
+  s: 'euiContextMenu--small',
+  m: null,
+};
+
+export const SIZES = keysOf(sizeToClassNameMap);
+
+export type MenuSize = 's' | 'm';
+
 export type EuiContextMenuProps = CommonProps &
   Omit<HTMLAttributes<HTMLDivElement>, 'style'> & {
     panels?: EuiContextMenuPanelDescriptor[];
     initialPanelId?: EuiContextMenuPanelId;
+    size?: MenuSize;
   };
```

Comment:
Here's a quick suggestion to reduce some of the duplication. Also, remember to only export the constants.
```suggestion
const sizeToClassNameMap = {
  s: 'euiContextMenu--small',
  m: null,
};

export const SIZES = keysOf(sizeToClassNameMap);

export type EuiContextMenuProps = CommonProps &
  Omit<HTMLAttributes<HTMLDivElement>, 'style'> & {
    panels?: EuiContextMenuPanelDescriptor[];
    initialPanelId?: EuiContextMenuPanelId;
    size?: typeof SIZES[number];
  };
```